examples/ModelComparisonWithSaliency.ipynb

```diff
@@ -0,0 +1,1843 @@
Ze9/+8Du985zvlPpIJJuvKGdB01pDshoZqtSJzbpcvr8kayL1jBiRngMm6ETRHvrincuZOM1j0F+T3ZOc0zYaKDJCYsCRTUxUTpPGxCexc2CUSToJASo+77bYU4GQGP/Cn78eLf/gl2L9/vwAmup5SAsv4jbe++b/irW//AyhqgExOg24VkCjMC/XTObj/D0DlPSHzSIB/RT7MdeQ1CpCIgb7BGl9BzgeMFuXhCXq2CsUw4PYced2NzS3ECg2hslhvbKBDwEu5YZQIaKXTM3n99qCPxT27cW75omS4hmRN4xD5HGNuVlEq5jGIOtypIpfVjERyJsl0rq6soZApIGvk0Q+A7iBGpzdAIZNBnhFIDueImWvJPMsQCws75XsIUlnAR0EohmKM8+E6arouhk7tbkcO3Rvf8Ebc+e47EQehzF7z/limIb8v5pl7y6FFBbt27Rb56tmz55Cl5LegI1/h88EX0x060vJXzxugnC+jtTVATsmjolbQ8ALUejHUyw2sjusIKBO2LJkx5/liLFLBzsDWdTRDyCiBoSSoFgtYX1uVZtNWM2WES+UythpdWDbfGxgZH8PU7DTq9XWsnFuDiwLaYQmhMQJ6DyN2MF61kC/FqK+fRNboY+f0JHKwUTIKCNn42liH0+qg57sSmYV2H+AMsm1CzZvYqnpwMj6sURO+34LBMM4wQrfvwfdiAfEyFxoTrJlwegkGfbohx9AUZoCaYm5GqT/VKDxHPFc8c3TQpnSeUlbOH0Mh8NJkfIJSbi4/51BFuhtHcqb92MX0/LicL86J816XyxU0G21hahM/NViiczcbRgSJbD7KzDjzkf3UrZqNQgLEnbvHsWvPNM4vnYLr9+VhYxg2DN2G43gCGAmy6bzMRiTZVoJYXnPoU6qeiLKEqhMaI0nHkfnKcub4dWxyUmJsijSZv+c4STZvitM1VDbRUpdoPkPkfftkxdm8ogEVn0+MrGHcDhtfOQGoIl03+Awj4KdChM+WlFnlWl16sj6seYf1znAFvk9WYHhYv09u1PBjDlfgO7ECd338vazFhc0URnPbMIdMEyWwhfII/CgRFo/mnE57E05nS+RhoZpGZJAZJagSICrS3wSRkgLQlDNNwQHfiAA2ZhYjZXlS0aWzaAS5/JUFCOV9kq/puAJQ84USNjeb8tnWNy5KASwyLiJZAortXwXvbmeXsvBLYx/ocpk62JJRUFgIKSwaszD10rbU1ReZJWcx5+cXpRgjW3DP3fdhdHQchqnjBS94IYIAqJRGxCTmE3fdhS9/9UsSY3H1geuQLZYA3cL41KTMxOWyNnzXoUmquHbS3fSvPvoRNLY2sWtxAddeew1mJibQbGzhnk/dLet3YN8+zM/No9/t4omjxyTWglmolM8GSYBEVeDHAWqjI4tdxlkAACAASURBVGhsXZZ1TQtWsiZkGFK5XRAq8Px0fdhUIJsIhV+nozZShKq5kiHIGTcCG86eEaTKvQo1vOY1r8X73ve+bUMSZj26qSRbisf0p4D8DMF/Ir8ni7V6+bIU0bxo3k8arHCeLTX9oeso51UdlMpF6CrzPAlmgIzJebtAooF279qDw4duQLvZRSFPl2ig3WI2Z8rM/+EfvAM/8aqfxMzMjLDjZM7On7skxfRb/sd/wVt/721S7EZxD7o1Ct0oyJ674kLLJknaMGABP5B7fvb0QVxez2Fm2sWhwxcFcH/2c9fAT4Birg+7dRpf+fJVeN6e38NNr34ZzImarNWg34efRHhq9QLCkQKMHWPAwEPjK0dRsrIwwgQZzUSr24WXhKhNTcCulbG6uoLGRh1Op42CbaFRr4sMPjtRRaFWRMbW0O9uYfXSRWEyN9cZ3aMhk7Wh56qojM3RcgmdrQbaZGRbTXj91KiKRTtniTkHKDN/iSIKgmfdcBgHr70GH/3oR0WSnEYTibAdr33ta/E///APhelnPBH3KsEC5w7JghPk8uvYAHE9X84x5b5JxkWkpoZZdHol88dZaYIP28xio+Fg3B7HntoudPwYxqUmlMubWJ3Q4St0kCU4sYVlpkmakaSGaW5xClu9PprrS7BUD1vryyiXitjY3MDI6CgK+QKa7T50IwcvTLBjcRGz83M4f/E8Lp05Bw85dKM8YFfl7JVzKhZnqpiarELVBzAsT2Slbj/BoBug16WMXUFAxi7yMWLbaB0/jebTZ6H4Pgb+AH7ZhzpuoWv2oGcTJIGPHmcmaSw3iGHRTMo0pUFGI6ReNxCpL+XlfG6algHbIuvopWc1jGXv8VkksUbgWESqOjBNoFrLo5Cz4PQcFAs1rK010O+mgJbPvsmZcWTKuoxk0Mxsc6spbr1kLsl8ZvWcuETTLIjz6NJElNEKjkKE4uKcynZV5AsmpufKMG1PwKnPOVKPM+8EypxvrYjDNl2GmQObRumYIhfnzKfTD9Fs9JAx89vMZgRd1WUmNfSlRSDPHnEbp5twkCoyGJPD2Vveiwh9zM6OY/eeBTz22BMC2hk5I4A/4Wx5Olfd6zkif6YZWCp7JvilmV2q5LkSQ8PmFVnUpWOXhzXvd6KQGL7HcAW+DSswPKzfhkUcvsRwBX5QVuDjf/s2wjsBUhwMYvFJ8wmaEUEtY3JqAbppieMnkgC9JgvrLeSLNiItBOeOCEoJUllME2SmgJNJE5TfEigRRybbDE+UBpTw67bnqvhQIsvDvwvJoFKSKIwCi74MbM73UeLGDrrD6IZAijSZPSUIo3RYI4ubvq6AVIJm/h3NQhFKniAlg1Acic5QYgOmlhdWNQhcYWJF+iZyO1Ukr73+QAw+aKz08le8QqI+KqUaLDuL+x/4NO779H3y/dPj86iOjUIzLZHo2oYuMj7Kh2MLMJnVqKp44NP3i6HM5Pgkrr/meuzfvQ+9ThOfuuvjyNsZVAsVRH6IRmMDCiVylREsrSxDNQ2EiAScDnxXADNZQrrlEnhR2inrR+dLK4O+xxVO3TcLxYzkHxKk9p0uRkaLCOMuBmRFOH
```

Comment:
(trying a second comment, not sure why the last one used the wrong line...)

The use of "in-line" doesn't make as much sense to me here (my brain goes to "in-line" code/compilation from C++ or other languages). Maybe:

```suggestion
    "This implementation is very similar to the `DRISEStack` algorithm, but allows us to define the granularity of our saliency maps in pixels whereas `DRISEStack` uses rigid square-grids that are deformed to cover an input image which results in less control in the shape of masked regions.\n",
```